app/routers/event.py

```diff
@@ -16,11 +17,17 @@
     raise_if_zoom_link_invalid,
 )
 from app.internal.emotion import get_emotion
+# TODO add this when the user system is merged (PR#195)
+# from app.internal.security.dependancies import (
+# current_user, CurrentUser)
 from app.internal.utils import create_model
 from app.routers.user import create_user
 
+
 TIME_FORMAT = '%Y-%m-%d %H:%M'
 
+PRIVATE = 'Private'
```

Comment:
Maybe we should have an `enum` for `PrivacyKinds`?

Reply:
Added in internal/privacy.py and updated in all relevant places